test/libevmasm/Optimiser.cpp

```diff
@@ -268,6 +268,26 @@ BOOST_AUTO_TEST_CASE(cse_double_shift_left_overflow)
 	}
 }
 
+BOOST_AUTO_TEST_CASE(cse_byte_ordering_bug)
+{
+	AssemblyItems input{
+			u256(31),
```

Comment:
```suggestion
		u256(31),
```
This only needs one level of indentation (same for the other lines)